client/gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar.scss

```diff
@@ -0,0 +1,40 @@
+.edit-post-pinned-plugins,
+.edit-post-more-menu__content {
+	.components-icon-button {
+		.jetpack-logo {
+			width: 20px;
+			height: 20px;
+		}
+	}
+}
+
+.edit-post-more-menu__content {
+	.components-icon-button {
+		.jetpack-logo {
+			margin-right: 4px;
+		}
+	}
+}
+
+.edit-post-pinned-plugins {
+	.components-icon-button {
+		&:not( .is-toggled ),
+		&:hover,
+		&.is-toggled,
+		&.is-toggled:hover {
+			.jetpack-logo,
+			.jetpack-logo__icon-circle,
+			.jetpack-logo__icon-triangle {
+				stroke: none !important;
+			}
+
+			.jetpack-logo__icon-circle {
+				fill: $green-jetpack !important;
+			}
+
+			.jetpack-logo__icon-triangle {
+				fill: $muriel-white !important;
```

Comment:
I think we want just `$white`. Our main shared colors assign `$white: $muriel-white`. I don't think we should rely on that variable here. Sorry if my other comment was unclear.

Reply:
Well, they both are `#fff` now, so 🤷‍♂️ . Updated in 8bd78e4805cae2f3a847223dd690c29cca95dfcb